supertokens_python/recipe/session/with_jwt/recipe_implementation.py

```diff
@@ -177,11 +181,51 @@ async def update_access_token_payload(
         )
 
         return await og_update_access_token_payload(
-            session_handle, new_access_token_payload, user_context
+            session_information.session_handle, new_access_token_payload, user_context
+        )
+
+    async def update_access_token_payload(
+        session_handle: str,
+        new_access_token_payload: Dict[str, Any],
+        user_context: Dict[str, Any],
+    ) -> bool:
+        # TODO: Node SDK allows non dict values of new_access_token_payload. But what should we do here?
+
+        session_information = await original_implementation.get_session_information(
+            session_handle, user_context
+        )
+        assert session_information is not None  # TODO: Is this a valid assumption?
+        return await jwt_aware_update_access_token_payload(
+            session_information, new_access_token_payload, user_context
+        )
+
+    async def merge_into_access_token_payload(
+        session_handle: str,
+        access_token_payload_update: Dict[str, Any],
+        user_context: Dict[str, Any],
+    ) -> bool:
+        session_information = await original_implementation.get_session_information(
+            session_handle, user_context
+        )
+        assert session_information is not None  # TODO: Is this a valid assumption?
```

Comment:
need to return false if this is not defined (have commented in the node SDK as well)

Reply:
done.